mantidproject/mslice
# Update MSlice start script and documentation

For development MSlice can be started by running 'mslicedevel.sh' or 'mslicedevel.bat'. Both relied on the script mantidpython in the Mantid install folder. This script is not part of Mantid installations anymore though.
Both 'mslicedevel.sh' or 'mslicedevel.bat' were adapted for this change and are calling start_mslice.py now directly and not via an additional script. Both new versions allow using either a conda installation for Mantid or a non-conda installation. There is a new environment file simplifying the required Mantid conda installation. 

When run outside an activated conda environment for Mantid, the scripts first try to activate such a conda environment and otherwise try to use a non-conda Mantid installation.

In addition, the online MSlice documentation and the readme.md for the MSlice repository have been updated.

**To test:**

Run the environment file and both scripts on Windows and Linux. Try different scenarios like conda/non-conda installation, problems with conda activate. For 'mslicedevel.bat', set CONDAPATH to the folder with your conda installation and try running it from file explorer by double-clicking on it.

Read the updated sections of the documentation and check that the changes make sense. Also, please check spelling and grammar in these sections ;-)

Fixes #814
